### DISCUSSION TITLE
No password confirmation when resetting a password

### DISCUSSION THREAD

**Discussion by susanthenerd** [plausible/analytics#2494] [Anything Else]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Self-hosting

### Describe the bug

When performing a password reset, there is no password confirmation form(aka typing two times the password)

### Expected behavior

Ask to confirm the password

**Comment by ukutaht** [comment]
Yes. This is an important feature that needs to be added.

**Comment by Vigasaurus** [comment]
I'd argue for pretty much all password entry, the more commonly accepted and user-preferred practice these days (and one which seems to cause the least user friction whilst still serving the purpose of reassurance and accuracy) is to use a show/hide password mechanism instead of the dated double-entry password fields, as this also allows for a consistent and useful capability on login pages as well as password creation/modification. 

One quick write-up I could find (it links out to a couple more studies & research near the bottom): https://designsmarts.co/show-password/
